xdsl/dialects/csl.py

```diff
@@ -1539,31 +1569,47 @@ class ParamOp(IRDLOperation):
         FscalesOp,
         FsubhOp,
         FsubsOp,
+        FuncOp,
+        GetColorOp,
+        GetFabDsdOp,
+        GetMemDsdOp,
+        ImportModuleConstOp,
+        IncrementDsdOffsetOp,
+        LayoutOp,
+        MemberAccessOp,
+        MemberCallOp,
         Mov16Op,
         Mov32Op,
         Or16Op,
+        ParamOp,
         PopcntOp,
+        ReturnOp,
+        RpcOp,
         Sar16Op,
+        SetDsdBaseAddrOp,
+        SetDsdLengthOp,
+        SetDsdStrideOp,
+        SetRectangleOp,
+        SetTileCodeOp,
+        SignednessCastOp,
         Sll16Op,
         Slr16Op,
         Sub16Op,
+        SymbolExportOp,
+        TaskOp,
         Xor16Op,
         Xp162fhOp,
         Xp162fsOp,
-        AddressOfOp,
-        SymbolExportOp,
-        RpcOp,
-        ParamOp,
     ],
     [
+        ColorType,
         ComptimeStructType,
+        DsdType,
         ImportedModuleType,
-        PtrKindAttr,
-        PtrConstAttr,
         PtrType,
-        DsdType,
-        ColorType,
         ModuleKindAttr,
+        PtrConstAttr,
+        PtrKindAttr,
         TaskKindAttr,
     ],
```

Comment:
All of this is just an alphabetical re-ordering. Makes finding things much easier when the lists get long